plugins/autoid/plugin.js

```diff
@@ -85,8 +87,28 @@
         editor.fire(EVENT_NAMES.ID_ADDED);
       }
 
+      function generateUniqueId() {
+          var uniqueId = '';
+          for (var i = 0; i < ID_LENGTH; i++) {
+            uniqueId += ALPHABET.charAt(Math.floor(Math.random() * ALPHABET.length));
```

Comment:
Is this concatenation method of string construction better/faster than appending to a list and then joining the characters together at the end?